admin-node-setup.sh

```diff
@@ -1,9 +1,15 @@
 #!/bin/sh
 # this script WILL BE RUN ON EVERY REBOOT
 
+# Clear the transactional update grains when booting up
+if [ -f /etc/salt/grains ]; then
+    sed -i -e 's|tx_update_reboot_needed:.*|tx_update_reboot_needed: false|g' /etc/salt/grains
```

Comment:
Is there any practical reason to prefer removal over setting it back to false? The decision to reset it to false rather than true was based on mimicking the logic in transactional-update itself.